helm-charts/explorer/templates/explorer-configmap.yaml

```diff
@@ -0,0 +1,112 @@
+
+          },
+          "peers": [
+            "peer0org1"
+          ],
+          "signedCert": {
+            "pem": "-----BEGIN CERTIFICATE-----\nMIICBjCCAaygAwIBAgIRAPW8giTkyghYAbO1LEBPbbQwCgYIKoZIzj0EAwIwWzEL\nMAkGA1UEBhMCVVMxEzARBgNVBAgTCkNhbGlmb3JuaWExFjAUBgNVBAcTDVNhbiBG\ncmFuY2lzY28xDTALBgNVBAoTBG9yZzExEDAOBgNVBAMTB2NhLm9yZzEwHhcNMjMw\nMjI3MDYzNDAwWhcNMzMwMjI0MDYzNDAwWjBfMQswCQYDVQQGEwJVUzETMBEGA1UE\nCBMKQ2FsaWZvcm5pYTEWMBQGA1UEBxMNU2FuIEZyYW5jaXNjbzEOMAwGA1UECxMF\nYWRtaW4xEzARBgNVBAMMCkFkbWluQG9yZzEwWTATBgcqhkjOPQIBBggqhkjOPQMB\nBwNCAATChVh6QCmBtI4/6S93XfEj7EDlz0TfISNDAB3rsyjj3T60QhRGczSQmFyE\nkx6nrrWNyXLsTe/gr4Cc0l//QKQpo00wSzAOBgNVHQ8BAf8EBAMCB4AwDAYDVR0T\nAQH/BAIwADArBgNVHSMEJDAigCDwWXLSHcu7lm4j8hzmjRtZV9TmXPv9GQiFJZJC\n2XpJgjAKBggqhkjOPQQDAgNIADBFAiEAu5qp7qKw6xYPl5miUgvNeGgGv6hVl17a\n1e7+DUExnZYCIGNsHEehl+VaRQ9c92vBStek6EgwqJ3dtXn3O7aEnT3f\n-----END CERTIFICATE-----\n"
+          }
+        }
+      },
+      "peers": {
+        "peer0org1": {
+          "tlsCACerts": {
+            "pem": "-----BEGIN CERTIFICATE-----\nMIICKTCCAc6gAwIBAgIRAJYfo9xEothMiUFEF6O4HdgwCgYIKoZIzj0EAwIwXjEL\nMAkGA1UEBhMCVVMxEzARBgNVBAgTCkNhbGlmb3JuaWExFjAUBgNVBAcTDVNhbiBG\ncmFuY2lzY28xDTALBgNVBAoTBG9yZzExEzARBgNVBAMTCnRsc2NhLm9yZzEwHhcN\nMjMwMjI3MDYzNDAwWhcNMzMwMjI0MDYzNDAwWjBeMQswCQYDVQQGEwJVUzETMBEG\nA1UECBMKQ2FsaWZvcm5pYTEWMBQGA1UEBxMNU2FuIEZyYW5jaXNjbzENMAsGA1UE\nChMEb3JnMTETMBEGA1UEAxMKdGxzY2Eub3JnMTBZMBMGByqGSM49AgEGCCqGSM49\nAwEHA0IABAJNnQijwM4kTL9Ja8J/dzdnfsHlU/wKBsyNcvN01Py+/au2LfNC70LO\nU8cl/y9zCI9iQotGtBQJBnWP/wsGxNijbTBrMA4GA1UdDwEB/wQEAwIBpjAdBgNV\nHSUEFjAUBggrBgEFBQcDAgYIKwYBBQUHAwEwDwYDVR0TAQH/BAUwAwEB/zApBgNV\nHQ4EIgQgNH4sKCOAIZPtYLyheNExU2f6hfAlv6CCyMITL5apnRIwCgYIKoZIzj0E\nAwIDSQAwRgIhALUakv5gqc+z99RWJhKIyrtYlX0Bf5+k6fn+6XfWWiH6AiEAl1bB\no8Axz4hJe/7BOEhOaG4NsrIokDwDt+hgBWurjA8=\n-----END CERTIFICATE-----\n"
+          },
+          "url": "grpcs://peer0-org1:7051"
+        }
+      }
+      }
+
+  config.json: |- 
+      {
+      "network-configs": {
+        "basic-network": {
+          "name": "basic-network",
+          "profile": "./basic-network.json"
+        }
+      },
+      "license": "Apache-2.0"
+      }
+
+
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: explorer-env
+  labels:
+    name: {{ $.Values.db.name }}
+    app.kubernetes.io/name: {{ $.Values.db.name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+  annotations: 
+    {{- toYaml .Values.annotations | nindent 6 }}     
+data:
+  DATABASE_HOST: {{ .Values.db.name }}
+  DATABASE_DATABASE: "fabricexplorer"
+  DATABASE_USERNAME: "hppoc"
+  DATABASE_PASSWD: "password"
+  LOG_LEVEL_APP: "debug"
+  LOG_LEVEL_DB: "debug"
+  LOG_LEVEL_CONSOLE: "info"
+  LOG_CONSOLE_STDOUT: "true"
+  DISCOVERY_AS_LOCALHOST: "false" 
```

Comment:
These properties should be dynamic.

Reply:
made above values are configurable in values.yaml file